starwars/src/App.js

```diff
@@ -33,6 +38,9 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        {this.state.starwarsChars.map ((character, index) =>  <Chars key=
+        {index} character={character} />)} 
```

Comment:
Use the issue that you had earlier with this line as a reminder of how to properly use the map function and use state.